money-monkey-web/components/CategoryPieChart.tsx

```
@@ -0,0 +1,136 @@
+            <p>{cat.name}</p>
+            <p>{formatMoney(sum.total_cents)}</p>
+          </div>
+        ),
+        color: cat.color,
+      });
+    }
+
+    return [
+      data,
+      config.reduce(
+        (acc, val) => ({ ...acc, [val!.key]: val }),
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)